examples/with-typescript-recommended/src/listeners/commands/chatInputCommands/chatInputCommandDenied.ts

```diff
@@ -0,0 +1,12 @@
+import type { ChatInputCommandDeniedPayload, Events } from '@sapphire/framework';
+import { Listener, UserError } from '@sapphire/framework';
+
+export class UserEvent extends Listener<typeof Events.ChatInputCommandDenied> {
+	public async run({ context, message: content }: UserError, { interaction }: ChatInputCommandDeniedPayload) {
+		// `context: { silent: true }` should make UserError silent:
+		// Use cases for this are for example permissions error when running the `eval` command.
+		if (Reflect.get(Object(context), 'silent')) return;
+
+		return interaction.channel?.send({ content, allowedMentions: { users: [interaction.user.id], roles: [] } });
```

Comment:
```suggestion
		return interaction.reply({ content, allowedMentions: { users: [interaction.user.id], roles: [] } });
```

Also add ephemeral, i.e. see https://github.com/favware/archangel/blob/main/src/listeners/commands/chatInputCommands/chatInputCommandDenied.ts

Also in general you should avoid `interaction.channel.send` and instead use replies an followups.